dashboard/app/models/course_offering.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: course_offerings
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
```

Comment:
Should we do the same key and display name set up we did for lesson group? So we don't run into the issues we have with lessons?

Reply:
Good idea